working-open.md

```diff
@@ -0,0 +1,46 @@
+# Working Open
+
+Hypha operates with democratic organizing principles informed by our [values](./values.md). As part of that we've developed a set of guidelines for how (and exactly what) and why we work in the open.
+
+## Why?
+
+- In order to share, provide opportunities for self- and collective- learning as we work alongside other new economy projects.
+- Not because a "default to open" is an ideal, but because it orients us toward negotiations a set of practices around our shared intentions.
+- Yet we recognize openly sharing might prevent frank and necessary discussion beneficial to our members, and so we want to clearly lay our our practices of consent and privacy
+
+# How and what?
+
+- We currently work with a [*in camera*](https://en.wikipedia.org/wiki/In_camera)/redaction request for meeting notes, where at any point members can request to stop taking notes, or redact and correct notes after they have been taken
+
+Legend:
+👀 - View only
+
+**Default open to all**:
+
+- Meeting notes (all hands, working groups) and final documents (handbook), on GitHub and other platforms 👀
+- Aggregated financal statements 👀
+- Project deliverables*, task tracking, meeting notes, and aggregated financial statements 👀
+
+* *This would include source repositories and code (e.g., on GitHub) and other process materials based on consensus on licensing and usage constraints.*
+
+**Default open to members**:
+
+- Meeting notes and documents (write-access)
+- Task tracking, project management (PM), and decision making (DM) tools (write-access)
+- Finances (budget, expenses, and cash flow projections) and audit records 👀
+- Potential opportunities and leads
+- Project proposal(s), statement(s) of work (in-progress/signed), in-progres deliverables, and finances (budget, expenses, and cash flow projections) 👀
+
+**Default closed to working groups and project members**:
+
+- Full personal/financial information of members
+- Personal/financial information involving external parties
+- Administration privileges including financal for services and infrastructure
+- Project sensitive data sets (especially all non-anonymized/identifiable) and critical infrastructure
+
+**Require consent before releasing**:
+
+- Audio, visual (photography) and video recordings
```

Comment:
Curious if the essence of this is visual representation of someone... As in, is it mostly likely that someone would allow a recording, so long as their face isn't present? Happy to defer this, but would like to eventually discuss framing this to model how folks usually conceptualize progressive consent to transparency :)

- audio recording of spoken word
- image or video representation of person

Reply:
Can you link to a resource on this? Super curious if there is intentionally some implicit degree difference by placing these as separate?

I was not thinking the essence was visual representation, with tri-council research ethics and in my previous work, the emphasis has been on ability to re-identify. Audio can be more identifiable than an image (in my experience). In terms of progressive consent-- the bullets you suggest doesn't reflect my experience // research consent forms and convos I've been in.

Happy to chat more but I think this might be a little premature?

Reply:
Hm. So your experience is not that images are perceived as more sensitive than audio? I've witnessed people avoid being on camera, but will ask questions from the audience knowing their voice is livestreamed. I'm not sure whether it meaningfully makes a difference in ethics, as both are perhaps equally likely to re-identify, but I've understood visual image to be more closely guarded than audio.

Where the rubber hits the road on this is that I can imagine someone consenting to audio of meeting being shared, but not video. I'm interested in the perceived baby-steps that might allow people to feel secure and in-control with sharing. Also, re-identificiation can happen from a verbatim transcript via [stylometric analysis](https://en.wikipedia.org/wiki/Stylometry), but I definitely have an assumption that most people feel a great deal more comfortable sharing a written transcript.

Sorry, no resources to share here, just my own perceptions.  Though Audrey Tang does have a [transparency policy](https://talk.pdis.nat.gov.tw/t/principles-for-handling-official-visits-to-digital-minister-audrey-tang/70) that favours at least one artifact be made public by default -- written, audio, or video -- but allows that to be chosen by person, presumably based on level of comfort and preference.

Reply:
I'm speaking from a (narrow) research context, but yeah wrt to photos v. audio. Video is a much higher bar and folks perception of that has many different facets. 

I'm going to resolve this here because I'v linked from #5 -- we'll definitely need to pick this up later, maybe with a project on deck? to help navigate